packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -76,7 +77,7 @@ async function collectInput(validProjects: Map<vscode.QuickPickItem, JDKVersion
     transformByQState.setTargetJDKVersion(JDKVersion.JDK17)
     await MultiStepInputFlowController.run(input => pickProject(input, state, validProjects))
     if (!state.project) {
-        throw new ToolkitError('No project selected', { code: 'NoProjectSelected' })
+        throw new ToolkitError('', { code: 'NoProjectSelected' })
```

Comment:
You'll see in a lot of places I deleted the message here, because it was showing a notification, but we already have notifications handled, so these were duplicates. Should I be throwing a regular Error instead perhaps?

Reply:
These shouldn't be empty strings. The handler should be catching these errors and then propagating the message to the user. Is there something I'm missing with how it works?

Reply:
Fixed to use `Error`